pkg/api/v1beta3/types.go

```diff
@@ -971,6 +971,16 @@ type NodeSystemInfo struct {
 	SystemUUID string `json:"systemUUID"`
 	// BootID is the boot-id reported by the node
 	BootID string `json:"bootID" description:"boot id is the boot-id reported by the node"`
+	// Kernel version reported by the node
+	KernelVersion string `json:"kernelVersion" description:"Kernel version reported by the node"`
+	// OS image used reported by the node
+	OsImage string `json:"osImage" description:"OS image used reported by the node"`
```

Comment:
Yes, I mean a linearized rollup of other package versions, like `container-vm-v20150112`. It's valuable information that may be necessary for upgrade.